helm/zenko/charts/mongodb-replicaset/.helmignore

```diff
@@ -0,0 +1,21 @@
+# Patterns to ignore when building packages.
```

Comment:
Could you, in the commit message, specify very specifically
- From which source these files were imported
- What the revision was of the foreign tree from which these files were copied
- Optionally, what the revision was of `stable/mongodb-replicaset` or whatever that was copied in that foreign tree

Reply:
Commit message has been amended